stack.yaml

```diff
@@ -20,6 +20,7 @@ packages:
 - core
 - db
 - lrc
+- lrc/tests
```

Comment:
Just looked at the `stack.yaml` file and we have both `util/test` and `crypto/tests`.

However, most of the packages had a `test` directory and only the `crypto` package has a `tests` directory. That means `lrc` should probably go with the flow and use `test` and an some stage we will fix `crypto` as well.